.github/workflows/operator-regression.yml

```diff
@@ -0,0 +1,68 @@
+name: Regression Testing Operator Integration
+
+on:
+  workflow_dispatch:
+
+env:
+  AGENT_TESTS: python nodejs java
+  #go dotnet --- both pending
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Create Kind cluster and local Docker registry
+        run:
+           bash test/operator/kind-with-registry.sh
+
+      - name: Create Test Images
+        run: |
+           for t in ${AGENT_TESTS[@]}
+           do
+             echo "Creating image for $t"
+             docker build -t $t-test-app test/operator/$t
+             docker tag $t-test-app localhost:5001/registry/$t-test-app
+             docker push localhost:5001/registry/$t-test-app
+           done
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: v3.11.2
+
+      - name: Install Operator Skeleton
+        run: |
+          kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.15.3/cert-manager.yaml
```

Comment:
This is needed only for apm tests right?

Reply:
WDYT shall we pin the versions of certmanager and azure/setup-helm@v4 or use the latest? In order those tests to catch any possible issues for the users as well?

Reply:
This is needed for the operator install unless the helm chart does that for you

Reply:
If we use the Onboarding steps, this won't be needed as `cert-manager` is disabled.